examples/intro_causal_graphs.py

```diff
@@ -0,0 +1,251 @@
+"""
+.. _ex-causal-graphs:
+
+====================================================
+An introduction to causal graphs and how to use them
+====================================================
+
+Causal graphs are graphical objects that attach causal notions to each edge
+and missing edge. We will review some of the fundamental causal graphs used
+in causal inference, and their differences from traditional graphs.
+"""
+
+# Authors: Adam Li <adam2392@gmail.com>
+#
+# License: BSD (3-clause)
```

Comment:
Let's keep these out. Authors are in git commits and the license has a dedicated file (which avoids that we get conflicting licenses in here).